packages/Path-View-Tests.package/SClassDiagramTest.class/instance/testToggleLabels.st

```diff
@@ -0,0 +1,21 @@
+as yet unclassified
+testToggleLabels
+
+	| dummyClassMorphs classDiagram classNameMorph elemShownBefore elemShownAfter elemShownAfter2 |
+	dummyClassMorphs := OrderedCollection newFrom: { SClassMorph on: testClass}.
+	classDiagram := klass new.
+
+	classDiagram instVarNamed: 'classMorphs' put: dummyClassMorphs.
+	classNameMorph := (dummyClassMorphs at: 1) submorphs at: 1.
+	elemShownBefore := classNameMorph submorphs size.
+
+	classDiagram toggleLabels.
+	classNameMorph := ((classDiagram instVarNamed: 'classMorphs') at: 1) submorphs at: 1.
+	elemShownAfter := classNameMorph submorphs size.
+
+	classDiagram toggleLabels.
+	classNameMorph := ((classDiagram instVarNamed: 'classMorphs') at: 1) submorphs at: 1.
+	elemShownAfter2 := classNameMorph submorphs size.
+
+	self assert: [elemShownAfter = (elemShownBefore - 1)].
+	self assert: [elemShownAfter = (elemShownAfter2 - 1)]
```

Comment:
This test checks both directions. Before the first 'toggleLabels' the 'classNameMorph' is visible. After the first toggling   classNameMorph contains one submorph fewer. This missing submorph comes back after the second toggling.

So I'm checking the toggling counting submorphs of the class name morph (one of them both dissapears)